kategory/src/main/kotlin/kategory/instances/NumberInstances.kt

```diff
@@ -39,4 +39,4 @@ object FloatMonoid : Monoid<Float>, Semigroup<Float> by SGFloat, GlobalInstance<
     override fun empty(): Float = .0f
 }
 
-private val SGFloat: Semigroup<Float> = NumberSemigroup(Float::plus)
\ No newline at end of file
+private val SGFloat: Semigroup<Float> = NumberSemigroup(Float::plus)
```

Comment:
Not sure why this is appearing as a change on my PR, but this instances were added by you @raulraja 😝 . Im not even using it :)